inst/htmljs/animint.js

```diff
@@ -27,13 +27,13 @@ var animint = function (to_select, json_file) {
             d[v_name] = parseInt(d[v_name]);
           } else if (r_type == "numeric") {
             d[v_name] = parseFloat(d[v_name]);
-          } else if (r_type == "factor" || r_type == "rgb" 
-		     || r_type == "linetype" || r_type == "label" 
+          } else if (r_type == "factor" || r_type == "rgb"
+		     || r_type == "linetype" || r_type == "label"
```

Comment:
please do not commit anything like this which shows up in the diff, but actually does not affect anything other than the formatting. (it makes it more difficult to see where the real changes are in the diff)